packages/zmarkdown/common.js

```diff
@@ -121,14 +121,12 @@ const zmdParser = (config, extraRemarkPlugins) => {
       }
     })
 
-  if (extraRemarkPlugins && extraRemarkPlugins.length > 0) {
-    for (const record of extraRemarkPlugins) {
-      if (!record.check || record.check(config)) {
-        if (record.option) {
-          mdProcessor.use(record.obj, record.option)
-        } else {
-          mdProcessor.use(record.obj)
-        }
+  for (const record of extraRemarkPlugins) {
+    if (!record.check || record.check(config)) {
+      if (record.option) {
```

Comment:
See https://github.com/zestedesavoir/zmarkdown/pull/268#discussion_r222430420

Reply:
It is changed.